rebeca-vargas/lesson-1/src/Car.java

```diff
@@ -0,0 +1,86 @@
+public class Car {
```

Comment:
You are missing performance Field, which should be a double that represents how much distance the car travels per gas unit